modules/eventing/pages/eventing-examples-rest-via-curl-get.adoc

```diff
@@ -0,0 +1,404 @@
+            "last_update_utc": now, "apiReqDateUtc": apiReqDateUtc,
+            "curl_success": false, "body_valid": false, "curl_time_ms": curl_time_ms
+        };
+    }
+    // rearm the timer
+    CreateRecurringTimer({ "id": context.id, "mode": "via_callback" })
+}
+
+function OnUpdate(doc, meta) {
+    // You would typically filter to mutations of interest 
+    if (doc.type !== 'recurring_timer') return;
+    if (doc.active === false) {
+        if (cancelTimer(RecurringTimerCallback, meta.id)) {
+            log('From OnUpdate: canceled active Timer, doc.active', doc.active, meta.id);
+        } else {
+            log('From OnUpdate: no active Timer to cancel, doc.active', doc.active, meta.id);
+        }
+    } else {
+        log('From OnUpdate: create/overwrite doc.active', doc.active, meta.id);
+        CreateRecurringTimer({  "id": meta.id, "mode": "via_onupdate" });
+    }
+}
+----
++
+After pasting, the screen appears as displayed below:
++
+image::ext_rest_via_curl_03_editor_with_code.png[,100%]
+** Click *Save*.
+** To return to the Eventing screen, click the '*< back to Eventing*' link (below the editor) or click *Eventing* tab.
+
+. The *OnUpdate* routine specifies that when a change occurs to data within the bucket, actions will be processed according to the field within the document.  First we ignore all documents that do not have a doc.type of "recurring_timer" this is the control document.  Next we use the field "active" to determine which action we take.  
+
+* If "active" is true we will create a series of daily Timers that will fire, however the fist timer will be approximately 30 seconds in the future from the time of deployment (for testing purposes).
+* If "active" is false we will cancel the existing Timer if any.
+* In the event a Timer created by this Function fires the callback *RecurringTimerCallback* executes and will write a new document with the a similar KEY (as the "source" bucket) but with "cur_" prepended into the "source" bucket.
+
+. From the *Eventing* screen, click *Deploy*.
+** In the *Confirm Deploy Function* dialog, select *Everything from the Feed boundary* option.
+** Click *Deploy Function*.
+
+. The Eventing function is deployed and starts running within a few seconds. From this point, the defined Function is executed on all existing documents and will also more importantly it will also run on subsequent mutations.
+
+== *Test 1: Create a Recurring Timer and allow the Timer to Fire and Rearm:*
+
+. Access the *Couchbase Web Console* > *Buckets* page and click the *Documents* link of the *source* bucket.
+** Edit the control document *recurring_timer::1* it should look like:
++
+----
+{
+  "type": "recurring_timer",
+  "id": 1,
+  "active": false
+}
+----
++
+Now change "active" to true to create a mutation, then Click *Save*.  This will create a mutation and then the Function will generate the first of a series of recurring Timers.  The control document is now:
```

Comment:
```suggestion
Change "active" to true then cick *Save*.  This will create a mutation and the Function will generate the first of a series of recurring Timers.  The control document is now:
```